packages/gatsby-transformer-cloudinary/create-remote-image-node.js

```diff
@@ -13,41 +13,41 @@ exports.createRemoteImageNode = async ({
   createNodeId,
   reporter,
 }) => {
-  if (!url) {
+  if (!reporter) {
     throw Error(
-      "`url` is a required argument. Pass the URL where the image is currently hosted so it can be downloaded by Cloudinary.",
+      "`reporter` is a required argument. It's available at `CreateNodeArgs.reporter`.",
+    );
+  }
+  if (!url) {
+    reporter.panic(
+      '`url` is a required argument. Pass the URL where the image is currently hosted so it can be downloaded by Cloudinary.',
     );
   }
   if (!parentNode) {
-    throw Error(
+    reporter.panic(
```

Comment:
I think we should replace throwing errors with reporter panics because they stop the build process. If any of these are missing, it can cause unexpected behavior. If we stop the build, we can alert developers to the problem and force the issue to be fixed more quickly.

Reply:
I agree with this